cmd/vic-machine/common/proxy.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/vic/pkg/flags"
+
+	"gopkg.in/urfave/cli.v1"
+)
+
+type Proxies struct {
+	HTTPSProxy *string
+	HTTPProxy  *string
+	IsSet      bool
+}
+
+func (p *Proxies) ProxyFlags(hidden bool) []cli.Flag {
+	return []cli.Flag{
+		// proxies
+		cli.GenericFlag{
+			Name:   "https-proxy, sproxy",
```

Comment:
hmm.. I just realize these two short name in this work, and tbh, I don't like this name as well. let me find some help on this